tests/unittests/tests-base64/tests-base64.c

```diff
@@ -303,13 +303,13 @@ static void test_base64_07_stream_decode(void)
 
 static void test_base64_08_encode_16_bytes(void)
 {
-    const int buffer_size = 16;
+    enum { buffer_size = 16 };
     unsigned char buffer[buffer_size];
```

Comment:
The Idea was to directly see the set size of the the buffer, instead of the need scrolling there and forth to check it.
It helped me a lot during the initial developing.